design/work_item_link.go

```diff
@@ -125,7 +125,6 @@ var workItemLinkList = JSONList(
 var _ = a.Resource("work_item_link", func() {
 	a.BasePath("/workitemlinks")
 	a.Action("show", showWorkItemLink)
```

Comment:
@hectorj2f actually, the `listWorkItemLinks()` function in the design is also used for the `workitem/relationships/links` endpoint, so I suggest we should keep it. Is it ok for you, @hectorj2f  ?